src/components/default-page-template.js

```diff
@@ -116,7 +116,7 @@ function DefaultPageTemplate({ children, fullScreen, headerActions, alerts }) {
       href: helpPage ? urls.getDocUrl(helpPage.documentKey, helpPage.documentSlug) : '',
       text: helpPage?.linkTitle,
       icon: QuestionOutlined,
-      permission: permissions.EDIT_SETTINGS,
+      permission: null,
```

Comment:
It might be that we will need specific pages for different types of users (registered and unregistered). I think the current ones are just serving registered users, so maybe we should put there permissions.EDIT_DOC or VIEW_DOCS?

Reply:
This is an error. You can always reach the help page from the footer link regardless if you are logged in or out.

Reply:
Oh, okay :).